models/intel/person-detection-0100/description/person-detection-0100.md

```diff
@@ -0,0 +1,55 @@
+# person-detection-0100
+
+## Use Case and High-Level Description
+
+This is a pedestrian detector for the Retail scenario. It is based on MobileNetV2
+backbone with two SSD heads from 1/16 and 1/8 scale feature maps and clustered
+prior boxes for 256x256 resolution.
+
+## Example
+
+![](./person-detection-0100.png)
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| AP                              | 82.52%                                    |
+| Pose coverage                   | Standing upright, parallel to image plane |
+| Support of occluded pedestrians | YES                                       |
+| Occlusion coverage              | <50%                                      |
+| Min pedestrian height           | 100 pixels (on 1080p)                     |
+| GFlops                          | 0.786                                     |
+| MParams                         | 1.817                                     |
+| Source framework                | Caffe*                                    |
+
+Average Precision (AP) is defined as an area under
+the [precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve.
+
+## Performance
+
+## Inputs
+
+Name: `input`, shape: [1x3x256x256] - An input image in the format [BxCxHxW],
+   where:
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order is BGR.
+
+## Outputs
+
+The net outputs blob with shape: [1, 1, N, 7], where N is the number of detected
+bounding boxes. For each detection, the description has the format:
+  [`image_id`, `label`, `conf`, `x_min`, `y_min`, `x_max`, `y_max`]
+  - `image_id` - ID of the image in the batch
+  - `label` - predicted class ID
```

Comment:
Since there only one class (isn't it?), would this value always set to one?

Reply:
there is the only class, but this is format of detection output layer

Reply:
I agree that it should be stayed as is for consistency with other models, changes in description can mislead users (speaking that it has difference from original DetectionOutput layer)